addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -319,7 +319,15 @@ $o-wslides-fs-side-width: 300px;
         border-width: 0 1px;
     }
 
-    .breadcrumb-item.active a, .breadcrumb-item a:hover {
+    .overflow {
```

Comment:
Can you target `breadcrumb-item` instead ? (not needed to create a new class as it's used only once)